blog/_posts/2020-05-20-technical-product-owner-for-when-agile-team-refactoring-rearchitecting.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "The Technical Product Owner: For When an Agile Team is Refactoring or Rearchitecting"
+layout: blog_post
+category: blog
+date: 2020-05-20
+description: How do you define your backlog when your team's work involves refactoring or is architectural in nature? The answer is to add a Technical Product Owner to your Scrum team.
+author: Calum Halcrow
+---
+
+## The Technical Product Owner: For When an Agile Team is Refactoring or Rearchitecting
+
+*How do you define your backlog when your team’s work involves refactoring or is architectural in nature?*
+
+At Rapid River we like to use Scrum. One thing we like about Scrum is the roles it places on software teams. The <a href="https://www.visual-paradigm.com/scrum/what-are-the-three-scrum-roles/" target="_blank">three key roles you’ll see on a Scrum team</a> are:
+
+* **Scrum Master**  - owns the process, ensuring it is followed and that it evolves with the needs of the team
+* **Development Team**  - own the implementation, ensuring the product is build correctly and robustly
+* **Product Owner**  - owns the product, and is in control of the backlog of work which the team implements.
+
+**The Typical Product Owner Role**
+
+We typically think of a Product Owner roles on a Scrum team in the context of building a product for end-users. Users who are perhaps paying a subscription to use the product. Users who are not employees of the company who own the software.
```

Comment:
Probably want `Product Owner's role` instead of `Product Owner roles`

Reply:
All fixed, thanks!